lib/languages/cpp.js

```diff
@@ -51,7 +51,14 @@ CppParser.prototype.parse_args = function(args) {
 };
 
 CppParser.prototype.get_arg_type = function(arg) {
-    return null;
+    if(arg === "...") {
```

Comment:
Adding a body for this function is unnecessary, because the type will be ignored anyway (see: [DocsParser.prototype.get_type_info](https://github.com/nikhilkalige/docblockr/blob/master/lib/docsparser.js#L97)).

But I would just leave it, maybe we need it some day..